ui/src/VoxettaApp.js

```diff
@@ -13,49 +13,18 @@ WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
 See the License for the specific language governing permissions and
 limitations under the License. */
 
+import {AudioRecorder} from './AudioRecorder';
 import {LitElement, html, css} from 'lit-element';
 
 export class VoxettaApp extends LitElement {
-    render() {
+    render() {  
         return html`
             <main>
                 Welcome to the Voxetta app
```

Comment:
Add your button

```javascript
import {VoxettaRecordButton} from './blah';

.............

return html`
 <main>
   <vox-record-button></vox-record-button>
 </main>
`;

Reply:
Why add the button component to this component and not in the html file?

Reply:
the index.html is supposed to be a template pretty much, with a main LitElement (in this case VoxettaApp.js) component comprising the whole app. doing this gives LitElement control over all our sub-components. if we continued to throw components into index.html instead of this component, any global styling, routing, SPA features etc. couldn't be handled by LitElement. this practice is the exact same in React (and I'd assume other frameworks as well) for the same reasons. it's understandably confusing if you haven't used a web framework before, but you'll soon see why this is how it works when we add things like the TOS and structure the UI together

Reply:
the index.html is supposed to be a template pretty much, with a main LitElement (in this case VoxettaApp.js) component comprising the whole app. doing this gives LitElement control over all our sub-components. if we continued to throw components into index.html instead of this component, any global styling, routing, SPA features etc. couldn't be handled by LitElement. this practice is the exact same in React (and I'd assume other frameworks as well) for the same reasons. it's understandably confusing if you haven't used a web framework before, but you'll soon see why this is how it works when we add things like the TOS and structure the UI together

Reply:
this ^

Reply:
Very insightful, I love it. Thank you!